### DISCUSSION TITLE
Can't find / create the api key in docker container

### DISCUSSION THREAD

**Discussion by Jegine** [ether/etherpad-lite#6429] [Q&A] [answered]
Hey everyone,

I'm trying to run etherpad as Docker container with the official etherpad image. To connect other applications to etherpad I need the api key from the etherpad instance.
I read through several resources and came across the APIKEY.txt file, which should be stored in the etherpad-lite folder. I can't find such a file in the entire container. I also tried to create a new file, but the etherpad instance doesn't seem to use it.
Is there an additional setting I have to make? In which exact location must the file be stored?

Thank you.

**Comment by claire-yq** [comment, marked as answer]
I believe the default authentication method is now OAuth2.0. If you want `APIKEY.txt` to be generated, try changing `"authenticationMethod"` to `"apikey"` in your `settings.json`.

**Reply by Jegine** [reply]
@sunshower7 Thank you very much. Setting the `authenticationMethod ` to `${AUTHENTICATION_METHOD:apikey} ` worked for me.
